qualtran/_infra/registers.py

```diff
@@ -206,6 +206,24 @@ def build(cls, **registers: int) -> 'Signature':
         """
         return cls(Register(name=k, bitsize=v) for k, v in registers.items() if v)
 
+    @classmethod
+    def build_from_dtypes(cls, **registers: QDType) -> 'Signature':
+        """Construct a Signature comprised of simple thru registers.
```

Comment:
distinguish from `build` in the first line. by editing this or the line for build or both

Reply:
Done.